src/server/templates/base.html

```diff
@@ -110,6 +166,9 @@ <h1 class="col-3 flex-grow-1">
       src="https://cdn.jsdelivr.net/npm/cookie-bar/cookiebar-latest.min.js?tracking=1&top=1"
     ></script>
 
+    <!-- Font Awesome -->
+    <script type="text/javascript" src="../static/js/fa.js"></script>
```

Comment:
This won't work on the server, you should just use `/static/js/fa.js`

Reply:
but shouldn't we also load that from a CND then?